options.html

```diff
@@ -101,16 +101,19 @@ <h4>K.K. Slider - Play Times</h4>
 						<div class="radio">
 							<input id="always-kk" name="enable-kk" type="radio"><label for="always-kk"><span><span></span></span>Play K.K. Slider songs at all times</label>
 						</div>
-						<h4>K.K. Slider - Version Choice</h4>
-						<div class="radio">
-							<input id="kk-version-live" name="kk-version" type="radio"><label for="kk-version-live"><span><span></span></span>Live</label>
-						</div>
-						<div class="radio">
-							<input id="kk-version-aircheck" name="kk-version" type="radio"><label for="kk-version-aircheck"><span><span></span></span>Aircheck</label>
-						</div>
-						<div class="radio">
-							<input id="kk-version-both" name="kk-version" type="radio"><label for="kk-version-both"><span><span></span></span>Both</label>
-						</div>
+						
+						<section id="kk-version-selection">
```

Comment:
![image](https://user-images.githubusercontent.com/18642415/72857175-ef31a280-3d10-11ea-913f-e2ab4358a942.png)
This creates a peculiar indentation.

Reply:
Ah good catch, should be fixed.